r/R/arrow-package.R

```diff
@@ -128,7 +128,7 @@ arrow_with_dataset <- function() {
   is_32bit <- .Machine$sizeof.pointer < 8
   is_old_r <- getRversion() < "4.0.0"
   is_windows <- tolower(Sys.info()[["sysname"]]) == "windows"
-  if (is_32bit && is_old_r && is_windows) {
+  if (on_old_windows()) {
```

Comment:
If we're moving this into a function, can we delete the variables above?

Reply:
Oh yep good catch